classes/Discount.php

```diff
@@ -224,8 +223,8 @@ public static function createOrderDiscount($order, $productList, $qtyList, $name
 		// create discount
 		$voucher = new Discount();
 		$voucher->id_discount_type = Discount::AMOUNT;
-		foreach ($languages as $language)
-			$voucher->description[$language['id_lang']] = strval($name).(int)($order->id);
+		foreach (Language::getIDs((bool)$order) as $id_lang)
```

Comment:
It's very odd, there is at least one more place like this. Argument should probably be `true` here, but I left it because I wasn't sure.